pkgs/development/python-modules/torch/default.nix

```diff
@@ -201,6 +201,8 @@ let
       MPISupport && cudaSupport && (mpi.cudatoolkit != cudaPackages.cudatoolkit);
     "Magma cudaPackages does not match cudaPackages" =
       cudaSupport && (effectiveMagma.cudaPackages != cudaPackages);
+    # torch requires hipblaslt, but the following is apparenly broken
+    "Rocm support is currently broken" = rocmSupport;
```

Comment:
Actually, the comment above the message could have been the message...

Reply:
Done, is it alright now ?

Reply:
What does" but the following is apparently broken" mean? The package seems to be missing in its entirety.